internal/apicodec/codec_v2.go

```diff
@@ -755,12 +778,20 @@ func (c *codecV2) decodeRegionError(regionError *errorpb.Error) (*errorpb.Error,
 	}
 
 	if errInfo := regionError.EpochNotMatch; errInfo != nil {
```

Comment:
Should we add some tests to cover this case? We could add some TODO here, could be done in the future.

Reply:
I have added some tests in codec_v2_test.go, PTAL